rootfs-builder/rootfs.sh

```diff
@@ -307,17 +327,46 @@ build_rootfs_distro()
 
 	mkdir -p ${ROOTFS_DIR}
 
+	# need to detect rustc's version too?
```

Comment:
May as well move this comment down a little to be with the `detect_rust_version` line?

Reply:
That maybe garbage from intermediate development.. sorry